drivers/src/lib.rs

```diff
@@ -10,4 +10,5 @@ pub mod plic;
 
 pub trait Console {
     fn write(&mut self, data: &str);
+    fn get_address_range(&self) -> (usize, usize);
```

Comment:
heeeeeeeeeuuuuuuuuu, I wouldn't put this in Console.

BUT, this is indeed an issue we are having.

Maybe it is time for a `Driver` trait, that, for now would just have a method to get an iterator over pages needed by the said driver.